src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -510,3 +582,28 @@ Commands:
     given ["--help"] |> yields message
     given ["-h"] |> yields message
     given ["-?"] |> yields message
+
+[<Fact>]
+let ``Shows help text for submit command`` () =
```

Comment:
The submit command itself won't be listed in the help function for the executable or in tab completion, but one can query the help for the submit explicitly - is that correct?

Reply:
Yep!